src/services/github-service.js

```diff
@@ -10,12 +10,16 @@ export class GitHubService {
     return fetch(`${this.baseUrl}/repositories?${query}`)
       .then((resp) => { return resp.json(); })
       .then((response) => {
-        return response.map((repo) => {
-          return {
-            id: repo.id,
-            name: repo.name
-          };
-        });
+        if (response && response.length) {
```

Comment:
In retrospect, it might be a bit "safer" / cleaner to avoid mutating `response` directly.   For example
```javascript
.then((response) => {
  const repos = [];

  if (response && response.length) {
    response.forEech((repo) => {
      repos.push({
        id: repo.id,
        name: repo.name
      })
    })
  }

  return repos;
})
```

Reply:
Thank you, @thescientist13,
- I will adjust using the "safer" method above. 
- I am getting up to speed with `lit-element` and can submit a "refactor PR" once I'm more familiar with the new (to me) techniques/approach used here.